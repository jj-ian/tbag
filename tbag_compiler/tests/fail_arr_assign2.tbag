/* Authors: Maria */
true {
        testvar();
        endgame;
}

func int testvar() {
        int [10] i;
        i [0] = "hello";
        return 0;
}
