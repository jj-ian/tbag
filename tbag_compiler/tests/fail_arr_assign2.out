Fatal error: exception Failure("Right hand side of assignment statement does
                    not match type of array")
